packages/pn-pa-webapp/tsconfig.json

```diff
@@ -32,6 +33,7 @@
     "node_modules",
     "**/__tests__/*",
     "**/__test__/*",
+    "src/test-utils.tsx",
```

Comment:
Of course it was only a confusion, I intended to exclude `test-utils.tsx`, but in fact I set it with the wrong path, since it lies inside `src/__test__`, and furthermore it is already excluded. Please remove!